frontend/src/components/MapView/LeftPanel/AnticipatoryActionPanel/HomeTable/index.tsx

```diff
@@ -257,34 +280,58 @@ function HomeTable() {
   const queryDate = getRequestDate(layerAvailableDates, selectedDate);
   const date = getFormattedDate(queryDate, DateFormat.Default) as string;
 
-  // TODO - LEVE is "MILD" and should be added as a new category, see Figma.
-
-  // TODO: implement NA and NY
-  // NA means that no proba is above the trigger for that district
-  // NY means that the district is not monitored yet (no rows for the district)
-  // const na = React.useMemo(() => [], []);
-  // const ny = React.useMemo(() => [], []);
-
   const headerRow: ExtendedRowProps = {
     id: -1,
     iconContent: null,
     windows: selectedWindow === 'All' ? AAWindowKeys.map(x => []) : [[]],
     header: selectedWindow === 'All' ? [...AAWindowKeys] : [selectedWindow],
   };
 
-  const dataForRows = React.useMemo(
-    () =>
-      rowCategories.map(r => ({
+  const dataForRows: {
```

Comment:
It feels like this whole processing logic, and `getDistrictData` in particular should be closer to where we load the data so that processing happens only once.

Let's try and do that as we transition to the new file

Reply:
Solved in #1170